index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com/css?family=Inter">
+  <link rel="stylesheet" href="style.css">
+  <title>Daniel Canon's Portfolio</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <div>
+        <ul class="shortcut-bar">
+          <li><a href="#projects">Portfolio</a></li>
+          <li><a href="#About-me">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
```

Comment:
Kindly I suggest you remove the menu list in the mobile version of your design, it should appear only in the desktop version

>### Your design
>
>
<img width="366" alt="Screenshot 2023-05-05 at 11 12 58 PM" src="https://user-images.githubusercontent.com/91237525/236595848-f201a1a8-be06-4b6d-8cad-40fa5eed93ff.png">